ddl/ddl_db_test.go

```diff
@@ -2385,6 +2385,28 @@ func (s *testDBSuite) TestAlterTableAddPartition(c *C) {
 	c.Assert(part.Definitions[4].LessThan[0], Equals, "maxvalue")
 	c.Assert(part.Definitions[4].Name, Equals, "p5")
 
+	s.tk.MustExec("drop table if t2")
```

Comment:
This is a syntax error, how can this pass test?

Reply:
It turns out that we cannot reuse the `s.tk` or the test doesn't fail.
We must call
```
tk := testkit.NewTestKit(c, s.store)
```
and use `tk` intead.